packages/libllvm/build.sh

```diff
@@ -64,64 +54,59 @@ TERMUX_PKG_HAS_DEBUG=false
 # cp: cannot stat '../src/projects/openmp/runtime/exports/common.min.50.ompt.optional/include/omp.h': No such file or directory
 # common.min.50.ompt.optional should be common.deb.50.ompt.optional when doing debug build
 
-termux_step_post_get_source() {
-	if [ "$TERMUX_PKG_QUICK_REBUILD" = "false" ]; then
-		mv clang-${TERMUX_PKG_VERSION}.src tools/clang
-		mv clang-tools-extra-${TERMUX_PKG_VERSION}.src tools/clang/tools/extra
-		mv lld-${TERMUX_PKG_VERSION}.src tools/lld
-		mv openmp-${TERMUX_PKG_VERSION}.src projects/openmp
-		mv compiler-rt-${TERMUX_PKG_VERSION}.src projects/compiler-rt
-		mv polly-${TERMUX_PKG_VERSION}.src tools/polly
-	fi
-}
-
 termux_step_host_build() {
 	termux_setup_cmake
 	termux_setup_ninja
 
-	cmake -G Ninja $TERMUX_PKG_SRCDIR
-	ninja -j $TERMUX_MAKE_PROCESSES clang-tblgen llvm-tblgen
+	cmake -G Ninja -DLLVM_ENABLE_PROJECTS='clang;lldb' $TERMUX_PKG_SRCDIR/llvm
+	ninja -j $TERMUX_MAKE_PROCESSES clang-tblgen lldb-tblgen llvm-tblgen
 }
 
 termux_step_pre_configure() {
 	if [ "$TERMUX_PKG_QUICK_REBUILD" = "false" ]; then
-		mkdir projects/openmp/runtime/src/android
-		cp $TERMUX_PKG_BUILDER_DIR/nl_types.h projects/openmp/runtime/src/android
-		cp $TERMUX_PKG_BUILDER_DIR/nltypes_stubs.cpp projects/openmp/runtime/src/android
+		mkdir openmp/runtime/src/android
+		cp $TERMUX_PKG_BUILDER_DIR/nl_types.h openmp/runtime/src/android
+		cp $TERMUX_PKG_BUILDER_DIR/nltypes_stubs.cpp openmp/runtime/src/android
 	fi
 
-	export LLVM_DEFAULT_TARGET_TRIPLE=$TERMUX_HOST_PLATFORM
+	export LLVM_DEFAULT_TARGET_TRIPLE=$CCTERMUX_HOST_PLATFORM
```

Comment:
Now including the Android API level in the default target triple, which was done to some extent already by the `Triple.h` patch, that's now removed as not needed after this change.